### PR TITLE
doc: update affiliation for myoungwon

### PR DESCRIPTION
**Pull request by myoungwon**
Signed-off-by: Myoungwon Oh <myoungwon.oh@samsung.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by myoungwon** [comment]
I moved to Samsung Electronics one and a half years ago, but my affiliation hasn't been changed correctly here. 
Can I update this?
(Samsung become premier member of Ceph foundation, and supports contribution)

**Review by tchaikov — APPROVED** [review]

**Comment by zdover23** [comment]
jenkins test make check arm64

File: doc/governance.rst [hunk 1]
```diff
@@ -62,7 +62,7 @@ Current CLT members are:
  * João Eduardo Luis <joao@suse.de>
  * Ken Dreyer <kdreyer@redhat.com>
  * Matt Benjamin <mbenjami@redhat.com>
- * Myoungwon Oh <omwmw@sk.com>
+ * Myoungwon Oh <myoungwon.oh@samsumg.com>
```
Comment (batrick) [inline doc/governance.rst:65]:
```suggestion
 * Myoungwon Oh <myoungwon.oh@samsung.com>
```

?

**Review by batrick — CHANGES_REQUESTED** [review]
Your signed-off-by is also has the typo (.gitconfig misconfiguration?)

**Comment by myoungwon** [comment]
@batrick Thanks. Done.

**Review by neha-ojha — APPROVED** [review]

**Review by batrick — APPROVED** [review]

**Pull request merged**